boards/openmote-b/include/board.h

```diff
@@ -77,6 +77,15 @@
 #define RF24_SWITCH_AT86RF215_TOGGLE (RF_SWITCH_PORT->DATA ^=  RF24_SWITCH_AT86RF215_MASK)
 /** @} */
 
+/**
+ * @name    RF CORE observable signals settings
+ * @{
+ */
+#define CONFIG_CC2538_RF_OBS_SIG_0_PCX  0   /* PC4 */
+#define CONFIG_CC2538_RF_OBS_SIG_1_PCX  1   /* PC6 */
+#define CONFIG_CC2538_RF_OBS_SIG_2_PCX  2   /* PC7 */
```

Comment:
The blinkenlights are not working on `openmote-b` but I can now use `at86rf215_subghz` together with `cc2538_rf`

Reply:
I didn't adapt the pins... hahha

Reply:
yeah I fixed that and something else that was mistaken in the init, but for some reason I currently only see the red led... dbugging

Reply:
There was a missing include as well.

Reply:
I know get flashy leds.